dockers/Screenshotter/screenshotter.js

```diff
@@ -343,16 +362,39 @@ function findHostIP() {
 //////////////////////////////////////////////////////////////////////
 // Take the screenshots
 
-var countdown = listOfCases.length;
+var countdown = listOfCases.length * modes.length;
 
 var exitStatus = 0;
 var listOfFailed = [];
 
+var doctypes = {
+    "no-quirks": "<!DOCTYPE html>",
+    "limited-quirks": '<!DOCTYPE HTML PUBLIC ' +
+        '"-//W3C//DTD HTML 4.01 Transitional//EN" ' +
+        '"http://www.w3.org/TR/html4/loose.dtd">',
+    "quirks": '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Draft//">',
+};
+
+var qabbr = {
+    "no-quirks": "nq",
+    "limited-quirks": "lq",
+    "quirks": " q",
```

Comment:
I (a) wanted them all the same width so that the test names align on output and (b) have the abbreviation letters match the words of the option. Can add a comment to that effect.